hnn_core/network_builder.py

```diff
@@ -85,14 +77,15 @@ def simulation_time():
     _PC.barrier()
 
     # these calls aggregate data across procs/nodes
-    _PC.allreduce(dp_rec_L2, 1)
-    # combine dp_rec on every node, 1=add contributions together
-    _PC.allreduce(dp_rec_L5, 1)
+    neuron_net.aggregate_dipoles()
+    _PC.allreduce(neuron_net.dipoles['L5_pyramidal'], 1)
+    _PC.allreduce(neuron_net.dipoles['L2_pyramidal'], 1)
+
     # aggregate the currents independently on each proc
     neuron_net.aggregate_currents()
     # combine neuron_net.current{} variables from each proc
-    _PC.allreduce(neuron_net.current['L5Pyr_soma'], 1)
-    _PC.allreduce(neuron_net.current['L2Pyr_soma'], 1)
+    _PC.allreduce(neuron_net.current['L5_pyramidal_soma'], 1)
```

Comment:
In line with the comment below, I'm confused why currents are recorded from just L5/L5 pyramidal soma. 

I'm thinking for #199 this variable will be indexed by gid.

Reply:
What does the GUI do? Is there a use-case for visualizing cell-level currents and/or Basket cell currents?

Reply:
There is certainly a use case, I believe GUI allows plotting currents of just L5/L2 pyramidal cells to compare against the voltage. But here it seems like the user can't access the recording as it doesn't get passed to either `Dipole` or `Network` once the simulation is run. 

Since this falls under the scope of the current recording PR I don't think you need to change anything now.